src/context/internal/uicontextresolver.cpp

```diff
@@ -88,7 +88,8 @@ UiContext UiContextResolver::currentUiContext() const
 
         ui::INavigationSection* activeSection = navigationController()->activeSection();
         if (!activeSection || activeSection->name() == NOTATION_NAVIGATION_SECTION) {
-            return context::UiCtxNotationFocused;
+            return notation->elements()->msScore()->inputState().staffGroup() == Ms::StaffGroup::TAB
+                   ? context::UiCtxTabNotationFocused : context::UiCtxStandardNotationFocused;
         }
 
         return context::UiCtxNotationOpened;
```

Comment:
I actually don't know when this line ever triggers and UiCtxNotationOpened is returned